aws/resource_aws_emr_security_configuration_test.go

```diff
@@ -109,7 +95,7 @@ func testAccCheckEmrSecurityConfigurationExists(n string) resource.TestCheckFunc
 }
 
 const testAccEmrSecurityConfigurationConfig = `
-resource "aws_emr_security_configuration" "foo" {
+resource "aws_emr_security_configuration" "test" {
```

Comment:
It is not an issue here, but I want to call out that there may be cases where these configurations might be shared across different resource tests. We've been encouraging folks not to reuse test configs and instead duplicate them to avoid such issues.